package.json

```diff
@@ -92,7 +92,7 @@
     "sinon": "^7.3.2",
     "spawn-wrap": "^2.0.0",
     "ts-node": "^9.1.1",
-    "typedoc": "^0.21.2",
+    "typedoc": "0.21.2",
```

Comment:
Any reason for this change?

Reply:
Sichao helped me resolve this. The reason is if not using the exact version, my change will build lots of case changes.